### DISCUSSION TITLE
How to add more Subjects to "Exlude items"

### DISCUSSION THREAD

**Discussion by PetrasekMartin** [phw198/OutlookGoogleCalendarSync#1971] [Q&A] [answered]
Hi, I want to add more subjects to "Subject that contains". For example: I have events with _dinner_ and _lunch_. How do I exlude both of those events?

**Comment by phw198** [comment, marked as answer]
It uses regular expressions, so you'd write something like `(dinner|lunch)`.

You can use eg www.regex101.com to test your pattern matching.
